apps/challenges/urls.py

```diff
@@ -40,7 +40,8 @@
     ),
     # `A-Za-z` because it accepts either of `all, future, past or present` in either case
     url(
-        r"^challenge/(?P<challenge_time>[A-Za-z]+)/(?P<challenge_approved>[A-Za-z]+)/(?P<challenge_published>[A-Za-z]+)$",
+        r"^challenge/time/(?P<challenge_time>[A-Za-z]+)/approved/"
+        "(?P<challenge_approved>[A-Za-z]+)/published/(?P<challenge_published>[A-Za-z]+)$",
```

Comment:
@gchhablani did you test EvalAI UI after this change? This will also need updating URLs in the frontend `challengeCtrl.js`. Changing this API must've broke the challenge dashboard page. Can you verify?